### PR TITLE
feat: t7 - chapter2-l2-t7-get-unique-files.sh

### PR DESCRIPTION
**Pull request by darliiin**
**Summary**
**Test task**
t7 - get only the filenames
Script should get a folder as an argument and return a list of all unique files with extensions but without the absolute path (including in the nested directories). only -f option is allowed for the use of find command, i.e.: find /tmp -type f. Which should find all files in all nested dirs and then process the output by using expansion.

name the file: `chapter2-l2-t7-get-unique-files.sh`

example of the call:

`chapter2-l2-t7-get-unique-files.sh /tmp/bazel`
```

main.sh.x.c
main.c
dummy.c
zutil.c
uncompr.c
trees.c
minigzip.c
infcover.c
example.c
inftrees.c
```
**my result**
`./chapter2-l2-t7-get-unique-files.sh ~/Загрузки/a `
**output**
```
photo_2023-03-20_16-00-58.jpg
Maven_spring_demo.zip
ideaIU-2022.3.3.tar.gz
google-chrome-stable_current_amd64.deb
dariyanalimova.p12
```

### CODE REVIEW

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# getting a list of all unique files with extensions but no absolute path
+
+# сhecking the number of arguments passed
+if [[ $# -ne 1 ]]; then
+  echo "You must specify one folder"
+  exit 1
+fi
+
+var=$1
```
Comment (kseniyashaydurova) [inline chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh:11]: Bad variable name

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# create an array
+file_name=()
```
Comment (kseniyashaydurova) [inline chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh:14]: Let's try to use `sort` and `uniq` bash commands to avoid having separate list

**Review by kseniyashaydurova — CHANGES_REQUESTED** [review]

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh (hunk 2 above)
Comment (darliiin) [reply chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh]: done

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh (hunk 1 above)
Comment (darliiin) [reply chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh]: done

**Review by kseniyashaydurova — APPROVED** [review]

**Pull request merged**